bayesian/api_v1.py

```diff
@@ -1141,6 +1142,43 @@ def post():
         return response, 200
 
 
+class CategoryService(ResourceWithSchema):
+    """Implementation of Dependency editor category service GET REST API call."""
+
+    method_decorators = [login_required]
+
+    @staticmethod
+    def get(runtime):
+        """Handle the GET REST API call."""
+        categories = defaultdict(lambda: {'pkg_count': 0, 'packages': list()})
+        gremlin_resp = get_categories_data(runtime)
+        response = {
+            'categories': dict(),
+            'request_id': gremlin_resp.get('requestId')
+        }
+        if 'result' in gremlin_resp and 'requestId' in gremlin_resp:
+            data = gremlin_resp.get('result')
+            if 'data' in data and data['data']:
+                for items in data.get('data'):
+                    category = items.get('category')
+                    package = items.get('package')
+                    if category and package:
+                        pkg_count = category.get('category_deps_count', [0])[0]
+                        _category = category.get('ctname', [''])[0]
+                        pkg_name = package.get('name', [''])[0]
```

Comment:
Do you want to continue even if  `_category` or `pkg_name` are empty ?

Reply:
`ctname` and `name` is the category name and package name values for these keys won't be empty if the category and package exist.